packages/discovery-provider/src/queries/get_notifications.py

```diff
@@ -32,6 +32,16 @@
   WHERE
     user_id =  :user_id
   AND is_current
+),
+distinct_specifiers as (
+    SELECT DISTINCT(specifier::int)
+    FROM notification 
+    WHERE :user_id = ANY(user_ids)
+    AND type = ANY(ARRAY['save', 'follow', 'repost'])
+),
+shadowbanned as (
+    SELECT get_shadowbanned_users(array_agg(specifier)) as specifier
```

Comment:
This is still using slow UDF?  Should we update to use pre-computed?

Reply:
this was left in by accident